src/components/Event.js

```diff
@@ -0,0 +1,14 @@
+function subscribe(eventName, listener) {
```

Comment:
This should not be in component. create this file in utils/

Reply:
Done